### PR TITLE
feat(ci): Add build of new agw container

### PR DESCRIPTION
**Pull request by quentinDERORY**
Signed-off-by: quentinDERORY <15911421+quentinDERORY@users.noreply.github.com>

## Summary

Publish new agw containers to artifactory

## Test Plan

On PR

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ardzoht — APPROVED** [review]
lgtm

File: .github/workflows/build_all.yml [hunk 1]
```diff
@@ -272,6 +272,103 @@ jobs:
           SLACK_ICON_EMOJI: ":heavy_check_mark:"
           SLACK_COLOR: "#00FF00"
           SLACK_FOOTER: ' '
+  agw-container-build:
```
Comment (arunuke) [inline .github/workflows/build_all.yml:275]: Is there a way to add multiple labels to the deployed images? If not labels, any other form of annotation? My hope is to be able to see images built over a number of days (ex: last 7 days) that we can then use for nightly builds and tests.

**Review by arunuke — APPROVED** [review]

File: .github/workflows/build_all.yml (hunk 1 above)
Comment (quentinDERORY) [reply .github/workflows/build_all.yml:275]: we could add labels to the docker images. What labels exactly would you want?

Comment (arunuke) [reply .github/workflows/build_all.yml:275]: Can we add commit ids? My hope is to identify which build this is based on.

Comment (quentinDERORY) [reply .github/workflows/build_all.yml:275]: It's already tagging with short sha and latest

**Pull request merged**